Extension/src/LanguageServer/configurations.ts

```diff
@@ -865,7 +860,7 @@ export class CppProperties {
             success = false;
         }
 
-        if (handleSquiggles && success) {
+        if (success) {
```

Comment:
so squiggles are handled now? Does this work for when edits are done to the UI and then user goes to json file?

Reply:
Oh I see, it only applies escapeForSquiggles on a copy of config and not the main copy.